identity/issuer/issuer.go

```diff
@@ -634,11 +637,32 @@ func (is *Issuer) Sign(string) (string, error) {
 	return "", fmt.Errorf("TODO")
 }
 
-// Sign signs a binary message by the kOp of the issuer.
+// SignBinary signs a binary message by the kOp of the issuer.
 func (is *Issuer) SignBinary(prefix, msg []byte) (*babyjub.SignatureComp, error) {
 	return is.keyStore.SignRaw(is.kOpComp, append(prefix, msg...))
 }
 
+// SignState signs the Identity State transition (oldState+newState) by the kOp of the issuer.
+func (is *Issuer) SignState(prefix []byte, oldState, newState *merkletree.Hash) (*babyjub.SignatureComp, error) {
```

Comment:
Since the function is called SignState, I think it should always use the `SigPrefixSetState`, so there's no need to pass it as argument.

Reply:
Right! done